packages/cdktf/lib/tfExpression.ts

```diff
@@ -234,8 +263,12 @@ class PropertyAccess extends TFExpression {
 }
 
 // eslint-disable-next-line jsdoc/require-jsdoc
-export function propertyAccess(target: Expression, args: Expression[]) {
-  return new PropertyAccess(target, args) as IResolvable;
+export function propertyAccess(
+  target: Expression,
+  args: Expression[],
+  forceNoBraces = false
```

Comment:
I don't see any code setting this to true – is this change still required? It seems like you solved the problem with the `resolveExpressionPart` change?

Reply:
Yeah that was old, removed it.